### PR TITLE
Update FAQ based on PR390

### PR DESCRIPTION
**Pull request by lilin90**
PTAL

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -409,13 +505,18 @@ There are [similar limits](https://cloud.google.com/spanner/docs/limits) on Goog
...
+- `admin show ddl`: to view the running DDL job
+- `admin show ddl jobs`: to view all the results in the current DDL job queue (including tasks that are running and waiting to run) and the last ten results in the completed DDL job queue
+
+#### The `column Show_db_priv not found` message is displayed when executing `grant SHOW DATABASES on db.*`.
 
-Note: The DDL cannot be cancelled unless it goes wrong.
+`SHOW DATABASES` is a global privilege rather than a database-level privilege. Therefore, you cannot grant this privilege to a database. You need to grant all databases:
+
+```
+`grant SHOW DATABASES on *.*`
```
Comment (queenypingcap) [inline FAQ.md]: Remove the ` mark from the sentence?

File: FAQ.md [hunk 2]
```diff
@@ -327,20 +365,72 @@ You can scale TiDB as your business grows.
...
+
 ### Migrate
 
 #### Can a MySQL application be migrated to TiDB?
 
 Yes. Your applications can be migrated to TiDB without changing a single line of code in most cases. You can use [checker](https://github.com/pingcap/tidb-tools/tree/master/checker) to check whether the Schema in MySQL is compatible with TiDB.
 
+#### Accidentally import the MySQL user table into TiDB and cannot login, how to restore?
+
+Restart the TiDB service, add the `-skip-grant-table=true` parameter in the configuration file. Login the cluster and rebuild using the following SQL statements:
```
Comment (queenypingcap) [inline FAQ.md]: rebuild using the following SQL statements: -> recreate the `mysql.user` table using the following statement:

File: FAQ.md [hunk 3]
```diff
@@ -327,20 +365,72 @@ You can scale TiDB as your business grows.
...
 
 The monitoring machine is recommended to use standalone deployment. It is recommended to use 8 core CPU, 32 GB+ memory and 500 GB+ hard disk.  
 
 #### The monitor can't show all the metrics.
 
 Check the time difference between the machine time of the monitor and the time within the cluster. If it is large, you can correct the time and the monitor will display all the metrics.
 
+#### How to configure to monitor Syncer status?
+
+Download [Syncer Json](https://github.com/pingcap/docs/blob/master/etc/Syncer.json) to Grafana, modify the Prometheus configuration file, add the following:
```
Comment (queenypingcap) [inline FAQ.md]: Download -> Download and import

Comment (queenypingcap) [reply FAQ.md]: modify the Prometheus configuration file, add the following:
-> edit the Prometheus configuration file and add the following content:

File: FAQ.md [hunk 4]
```diff
@@ -268,6 +293,19 @@ To solve this problem, use the [store delete](https://github.com/pingcap/pd/tree
 
 The RocksDB compresses the key.
 
+#### TiKV master and slave use the same compression algorithm, why the results are different?
+
+Currently, some files of TiKV master have a higher compression rate, which depends on the underlying data distribution and RocksDB implementation. It is normal that the data fluctuates occasionally. The underlying storage engine adjusts data as needed.
+
+#### What are the features of TiKV block cache?
+
+TiKV has the Column Falimies feature of RocksDB. By default, the KV data is eventually stored in the 3 CFs (default, write and lock) within RocksDB.
```
Comment (queenypingcap) [inline FAQ.md]: TiKV has the Column Falimies feature of RocksDB. -> TiKV implements the Column Family feature of RocksDB.

File: FAQ.md [hunk 5]
```diff
@@ -268,6 +293,19 @@ To solve this problem, use the [store delete](https://github.com/pingcap/pd/tree
 
 The RocksDB compresses the key.
 
+#### TiKV master and slave use the same compression algorithm, why the results are different?
+
+Currently, some files of TiKV master have a higher compression rate, which depends on the underlying data distribution and RocksDB implementation. It is normal that the data fluctuates occasionally. The underlying storage engine adjusts data as needed.
```
Comment (queenypingcap) [inline FAQ.md]: data fluctuates occasionally -> data size

File: FAQ.md [hunk 6]
```diff
@@ -226,17 +235,33 @@ Yes. Besides TiKV, TiDB supports many popular standalone storage engines, such a
...
+- A long time of not accessing can also lead to this error, usually caused by the tcp timeout. If not used for a long time, the tcp is killed by the operating system. 
+
+#### Can I use S3 as the backend storage in TiDB?
+
+No. Currently, TiDB only supports the distributed storage engine and the Goleveldb/Rocksdb/Boltdb engine.
+
+Does TiDB support the following DDL?
+
+```
+`CREATE TABLE ... LOCATION "s3://xxx/yyy"`
```
Comment (queenypingcap) [inline FAQ.md]: Please remove the ` mark.

File: FAQ.md [hunk 7]
```diff
@@ -226,17 +235,33 @@ Yes. Besides TiKV, TiDB supports many popular standalone storage engines, such a
 
 In RocksDB.
 
-#### Why it is very slow to execute DDL statements sometimes?
+#### Why it is very slow to run DDL statements sometimes?
 
-In the TiDB cluster, the DDL statements are executed serially, not concurrently. You can use the `admin show ddl` statement to view running DDL statements and use the `admin show ddl jobs` statement to view the DDL in history and in the queue. 
+Possible reasons:
+
+- When you run multiple DDL statements together, it is slower when running the later few DDL statements. This is because the DDL statements are executed serially in the TiDB cluster.
```
Comment (queenypingcap) [inline FAQ.md]: When you -> If you

**Comment by lilin90** [comment]
PTAL @queenypingcap

**Pull request merged**